precompile/modules/registerer.go

```diff
@@ -30,6 +30,11 @@ var (
 			Start: common.HexToAddress("0x0300000000000000000000000000000000000000"),
 			End:   common.HexToAddress("0x03000000000000000000000000000000000000ff"),
 		},
+		// Individual reserved address for p256verify precompile
+		{
+			Start: common.HexToAddress("0x0000000000000000000000000000000000000100"),
+			End:   common.HexToAddress("0x0000000000000000000000000000000000000100"),
+		},
```

Comment:
This reserved range is for stateful precompile modules, so this does not apply stateless precompiles. We should not reserve a range for this, otherwise stateful precompiles can register in this reserve.